.github/workflows/wheels.yml

```diff
@@ -98,6 +98,21 @@ jobs:
           name: ${{ matrix.python[0] }}-${{ startsWith(matrix.buildplat[1], 'macosx') && 'macosx' || matrix.buildplat[1] }}
           path: ./wheelhouse/*.whl
 
+      # Used to push the built wheels
+      # TODO: once Python 3.11 is available on conda, de-dup with
+      # setup python above
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          # Really doesn't matter what version we upload with
+          # just the version we test with
+          python-version: '3.8'

```

Comment:
Do we want to use mamba? I think it's just adding a parameter here

Reply:
```suggestion
          python-version: '3.8'
          mamba-version: "*"
```